### DISCUSSION TITLE
[GitHub Actions]Pull requests always include some outdated checks

### DISCUSSION THREAD

**Discussion by morphinewan** [community/community#26104] [Actions]
<p>I updated my github actions workflow files to new yaml format. And found out every pull request will contains a expected check that is outdated and not exists in current workflow file. This cause the pull request’s status stuck in status “Some checks haven’t completed yet” like below.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/136bb54958371d52e52d02980a080638a739025d.png" data-download-href="/uploads/short-url/2LNPNZELX4gfcx0eHJ4LB4oYjG5.png?dl=1" title="屏幕快照 2019-08-27 08.24.42.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/458887/181095444-f89f4d20-7e66-4efb-b2d3-c6080c46d329.png" alt="屏幕快照 2019-08-27 08.24.42.png" data-base62-sha1="2LNPNZELX4gfcx0eHJ4LB4oYjG5" width="690" height="375" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/136bb54958371d52e52d02980a080638a739025d_2_690x375.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/136bb54958371d52e52d02980a080638a739025d_2_1035x562.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/136bb54958371d52e52d02980a080638a739025d_2_1380x750.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/136bb54958371d52e52d02980a080638a739025d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">屏幕快照 2019-08-27 08.24.42.png</span><span class="informations">1664×906 110 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The “Eslint Jest” check is removed a long time before and does not exist any more. But every pull request created now will contains it and will be expectd its status.How to fix that? </p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/morphinewan">@morphinewan</a>,</p>
<p>Thank you for being here and apologies for the delay in response! Looks like perhaps this is a protected branch and you’d need to disable the required checks on Eslint Jest. </p>

**Comment by morphinewan** [comment]
<p>Thank you for your reply. But my problem is “ESLint Jest” task does not exists any more in fact. How can I disable the required checks of a non-exists check?</p>

**Comment by morphinewan** [comment]
<p>For more information.</p>
<p>I created the check named “ESLint Jest” for unit testing purpose. Then I renamed the check to chinese name “单体测试” after I upgraded to new yaml format workflow. And then every pull request will contains two checks. And the “ESLint Jest” check will never end or be excuted. And there is no logs about this outdated check in “Action” page.</p>
<p>It seems some kind of bug of Github Action.</p>

**Comment by Legion2** [comment, marked as answer]
<p>You can remove the outdated check in the protected branch settings of the repository settings.</p>

**Comment by morphinewan** [comment]
<p>Thanks, problem is solved.</p>
